### PR TITLE
Update python changelog

### PR DESCRIPTION
**Pull request by benjeffery**
@nspope @petrelharp

I've reviewed the diff of tskit and found that some new documented methods were missing changelogs. Have I got the details right here? Thanks!

### CODE REVIEW

File: python/CHANGELOG.rst [hunk 1]
```diff
@@ -59,6 +60,24 @@
   which assigns a string to node and mutation symbols, commonly shown on mouseover. This
   can reduce label clutter while retaining useful info (:user:`hyanwong`, :pr:`3007`)
 
+- Added ``distance_between`` that calculates the total distance between two nodes in a tree.
+  (:user:`Billyzhang1229`, :pr:`2771`)
+
+- Added ``genetic_relatedness_matrix`` method to compute
+  pairwise genetic relatedness between sample sets.
+  (:user:`jeromekelleher`, :user:`mmosmond`, :user:`petrelharp`, :pr:`2823`, :pr:`1623`)
```
Comment (jeromekelleher) [inline python/CHANGELOG.rst:68]: I think this was just pr 2823, and mmosmond didn't really contribute

**Review by jeromekelleher — APPROVED** [review]
LGTM, one minor thing

File: python/CHANGELOG.rst [hunk 2]
```diff
@@ -59,6 +60,24 @@
...
+
+- Added ``genetic_relatedness_vector`` method to compute product of genetic relatedness
+  matrix and weight vector.
+  (:user:`petrelharp`, :pr:`2980`)
+
+- Added ``pair_coalescence_counts`` method to calculate coalescence events per node or time
+  interval, and ``pair_coalescence_quantiles`` method to estimate quantiles of pair
+  coalescence times using empirical CDF inversion.
```
Comment (nspope) [inline python/CHANGELOG.rst:76]: also:

and ``pair_coalescence_rates`` method to estimate instantaneous rates of pair coalescence within time intervals from the empirical CDF (:pr:`2985`)

**Comment by nspope** [comment]
LGTM aside from one missing PR -- thanks @benjeffery

**Comment by benjeffery** [comment]
Codecov losing the plot and claiming I've reduced coverage by 3% by editing the changelog. Very tempted to move off it!

**Pull request merged**